src/System Application/App/Filter Tokens/src/FilterTokensImpl.Codeunit.al

```diff
@@ -387,6 +387,7 @@ codeunit 58 "Filter Tokens Impl."
     local procedure ResolveTimeToken(var TimeFilter: Time; TimeToken: Text) Handled: Boolean
     begin
         TimeToken := DelChr(TimeToken); // Deletes all spaces in TimeToke
```

Comment:
If we are already here, we could also ...
```suggestion
        TimeToken := DelChr(TimeToken); // Deletes all spaces in TimeToken
```

Reply:
Yeah, I saw that one... Usually I try to keep my PRs very clean and 100% related to the problem I'm trying to fix, but in this case I guess I could have made an exception ;-) Oh well...